components/de_amp/browser/test/de_amp_util_unittest.cc

```diff
@@ -116,20 +121,19 @@ TEST(DeAmpUtilUnitTest, NegativeDetectAmp) {
       "</head>\n"
       "<body></body>\n"
       "</html>";
-  CheckFindCanonicalLinkResult("", body, false);
+  CheckFindCanonicalLinkResult("", body, false, false);
 }
 
 TEST(DeAmpUtilUnitTest, DetectAmpButNoCanonicalLink) {
   // Put AMP attribute in a different tag than html
```

Comment:
The comment looks unrelevent now.

Reply:
Why?

Reply:
because now we have `amp` in `html` too.